scripts/gen_lib.lua

```diff
@@ -0,0 +1,71 @@
+lpeg = require('lpeg')
```

Comment:
>  It still does not tell whether helpers.lua is of “lua helpers.lua --do-something” type or a lua module.

I think it is obvious enough. But if conformity to tests are irrelevant,  we can`generators/gen_xx.lua` and `generators/lib.lua`. (though  `generators/lib/xx.lua` is better if already expect multiple library files). But I see little point to `generators/bin`, generators should just be in `generators/`.